oauthenticator/github.py

```diff
@@ -215,8 +231,68 @@ async def authenticate(self, handler, data=None):
                     auth_state['github_user']['email'] = val['email']
                     break
 
+        if self.fetch_teams:
+            if 'read:org' not in self.scope:
+                # This means the 'read:org' scope was not set, and we can't fetch teams
+                self.log.error(
+                    'read:org scope is required for fetch_teams functionality to work'
+                )
```

Comment:
Should it be an error or a warning? I was thinking a warning is more appropriate since a user will successfully be authenticated etc.

Reply:
I think this should be an error though, as the software is not functioning as it is configured to.